codec-http/src/main/java/io/netty/handler/codec/http/multipart/DefaultHttpDataFactory.java

```diff
@@ -110,8 +115,10 @@ public void setMaxLimit(long maxSize) {
 
     /**
      * @return the associated list of Files for the request
+     *
+     * Visible for testing.
      */
-    private List<HttpData> getList(HttpRequest request) {
+    List<HttpData> getList(HttpRequest request) {
```

Comment:
it would be ideal if we could test using the public API of the class instead of depending on the internals (which are subject to change over time).